sql/catalyst/src/main/scala/org/apache/spark/sql/catalyst/expressions/stringExpressions.scala

```diff
@@ -3333,22 +3333,45 @@ case class FormatNumber(x: Expression, d: Expression)
 
 /**
  * Splits a string into arrays of sentences, where each sentence is an array of words.
- * The 'lang' and 'country' arguments are optional, and if omitted, the default locale is used.
+ * The `lang` and `country` arguments are optional, their default values are all '',
+ *  - When they are omitted:
+ *    1. If they are both omitted, the `locale(language='', country='')` is used.
```

Comment:
This is `Locale.ROOT`, right? Let's leave a few words for non-Java users like `the base locale of all locales, and is used as the language/country neutral locale for the locale sensitive operations.`

Reply:
Yes, this is Local.ROOT
<img width="1167" alt="image" src="https://github.com/user-attachments/assets/78d6f1d1-db0e-430b-ae7c-20f680a58934">

Let me update it.

Reply:
Updated.